StRoot/StJetMaker/StJetSkimEventMaker.cxx

```diff
@@ -242,6 +242,22 @@ Int_t StJetSkimEventMaker::Make()
 		mEvent->setVpdZvertex(muDst->btofHeader()->vpdVz());
 		mEvent->setVpdEastHits(muDst->btofHeader()->numberOfVpdHits(east));
 		mEvent->setVpdWestHits(muDst->btofHeader()->numberOfVpdHits(west));
+
+                // Start: bassam
+                mEvent->setNTzero(muDst->btofHeader()->nTzero());
+                mEvent->setNTzeroCan(muDst->btofHeader()->nTzeroCan());
+                mEvent->setTCanFirst(muDst->btofHeader()->tCanFirst());
+                mEvent->setTCanLast(muDst->btofHeader()->tCanLast());
+                // filling in pile-up information from the VPD
+                mEvent->setNVpdEGoodHits(muDst->btofHeader()->vpdEGoodHits());
+                mEvent->setNVpdWGoodHits(muDst->btofHeader()->vpdWGoodHits());
+                mEvent->setEarliestVpdEHit(muDst->btofHeader()->earliestVpdEHit());
+                mEvent->setEarliestVpdWHit(muDst->btofHeader()->earliestVpdWHit());
+                mEvent->setClosestVpdEHit(muDst->btofHeader()->closestVpdEHit());
+                mEvent->setClosestVpdWHit(muDst->btofHeader()->closestVpdWHit());
+                mEvent->setLatestVpdEHit(muDst->btofHeader()->latestVpdEHit());
+                mEvent->setLatestVpdWHit(muDst->btofHeader()->latestVpdWHit());
+                // End: bassam 
```

Comment:
Check indentation. You may be using tabs instead of recommended spaces. See the STAR coding guidelines.